src/Form/Select.svelte

```diff
@@ -1,14 +1,26 @@
 <script>
+  import Icon from "../Icons/Icon.svelte";
+  import { createEventDispatcher } from "svelte";
+
   export let value = "";
   export let name = undefined;
   export let thin = false;
   export let secondary = false;
   export let outline = false;
   export let disabled = false;
-  import Icon from "../Icons/Icon.svelte";
+  export let editable = false;
+  
+  const dispatch = createEventDispatcher();
+
+  const updateValue = type => e => {
+    value = e.target.value;
+    dispatch("blur", e)
```

Comment:
This is nice, although should it be `dispatch(type, e)`

Reply:
yes, good one cheers